libdrgn/Makefile.am

```diff
@@ -18,6 +18,8 @@ libdrgnimpl_la_SOURCES = binary_search_tree.h \
 			 hash_table.h \
 			 internal.c \
 			 internal.h \
+			 kdump_reader.c \
```

Comment:
Instead of having it here, we can add `libdrgnimpl_la_SOURCES += kdump_reader.c` in the `if WITH_LIBKDUMPFILE` block, that way you don't need to put the big ifdef in `kdump_reader.c`.